### PR TITLE
ci/ext_proc: Temporarily disable flaking test

### PR DESCRIPTION
**Pull request by phlax**
Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by phlax** [comment]
tracking ticket to re-enable the test is here https://github.com/envoyproxy/envoy/issues/35281

**Comment by phlax** [comment]
cc @tyxia

**Review by KBaichoo — DISMISSED** [review]

**Review by KBaichoo — APPROVED** [review]

File: test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc [hunk 1]
```diff
@@ -1599,7 +1599,8 @@ TEST_P(ExtProcIntegrationTest, GetAndSetHeadersAndTrailersOnResponse) {
 // Test the filter using the default configuration by connecting to
 // an ext_proc server that tries to modify the trailers incorrectly
 // according to the header mutation rules.
-TEST_P(ExtProcIntegrationTest, GetAndSetTrailersIncorrectlyOnResponse) {
+#TODO(@tyxia): reenable this test (see https://github.com/envoyproxy/envoy/issues/35281)
```
Comment (tyxia) [inline test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc:1602]: Do you just need `TODO(tyxia)` ?

At least that is what i used in the past :)

Comment (phlax) [reply test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc:1602]: probs that would have worked - hoping the format issue is resolved now tho

Comment (phlax) [reply test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc:1602]: argh - still breaking 8(

Comment (KBaichoo) [reply test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc]: could just do `TODO(tyxia) see #35281`

Comment (phlax) [reply test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc]: i think the problem was i was using wrong comment prefix - ie `#` and not `//`

**Review by KBaichoo — APPROVED** [review]

**Pull request merged**